src/site/content/en/blog/local-fonts/index.md

```diff
@@ -324,12 +293,6 @@ The Chrome team has designed and implemented the Local Font Access API using the
 defined in [Controlling Access to Powerful Web Platform Features][powerful-apis], including user
 control, transparency, and ergonomics.
 
-### User control
-
-Access to a user's fonts is fully under their control and will not be allowed unless the
```

Comment:
This bit can stay as it's possible for web developers to check permission status with code below:
```js
const status = await navigator.permissions.query({name: 'local-fonts'});
if (status.granted) { console.log('local fonts access has been granted'); }
```

Reply:
https://w3c.github.io/permissions-registry/#registry-table-of-standardized-permissions only has two permissions where we know there are more (even standard ones like "camera" and "microphone").

https://web.dev/async-clipboard/, https://web.dev/media-recording-video/, https://web.dev/periodic-background-sync/, https://web.dev/multi-screen-window-placement/ and more uses "navigator.permissions.query" to show how to query permission status. I think you're okay.

Reply:
Leaving it in then.